datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -907,7 +907,12 @@ type AspectRenderSpec {
 """
 Additional read only properties about a Dataset
 """
-type DatasetProperties {
+type DatasetProperties implements Aspect {
+    """
+    The logical version of the schema metadata, where zero represents the latest version
```

Comment:
docs are copy pasted, so `schema metadata` appears and should be replaced by the specific aspect name.

Reply:
Whoops, will fix